src/docs/reference/support-policy.md

```diff
@@ -74,6 +74,8 @@ The table below describes recent versions of Stencil and the version of TypeScri
 
 | Stencil Version | TypeScript Version |
 |:---------------:|:------------------:|
+|     v2.21.0     |       v4.9.3       |
+|     v2.20.0     |       v4.8.4       |
```

Comment:
Did we never update this for 4.8?

Reply:
We did, I just haven't rebased this since the 4.8 stuff landed ~1 hr ago

Reply:
Should be fixed now 👍 https://github.com/ionic-team/stencil-site/pull/939/commits/6ab1e7f6b80848c676f03af43551778bc2103c20